src/state-summary/state-card-display.html

```diff
@@ -75,9 +75,9 @@
           stateObj._stateDisplay = window.hassUtil.formatDateTime(date);
         }
       } else if (domain === 'zwave' && ['initializing', 'dead'].includes(stateObj.state)) {
-        stateObj._stateDisplay = haLocalize(`state.${domain}`, stateObj.state, 'query_stage', stateObj.attributes.query_stage);
```

Comment:
Lets change ${domain} to zwave here and two lines below - it would be more readable